README.md

```diff
@@ -1,36 +1,82 @@
-# n-gage
+# n-gage [![CircleCI](https://circleci.com/gh/Financial-Times/n-gage.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://github.com/Financial-Times/n-gage)
 
-<a href="https://docs.google.com/forms/d/e/1FAIpQLSf5InA7UJK9yNBCzidFKI_WNkfbl6of1eRlIACRspGXUcBx8A/viewform?usp=pp_url&entry.78759464=n-gage" target="_blank"><img src="https://i.imgur.com/UmScdZ4.png" alt="Yak button" border="0" align="right" width="150" title="Report a yak shaving incident for this repository"></a>
+`n-gage` is in every FT.com project and gives a standard set of `make` tasks and the `ngage` CLI to help set up, build and deploy projects.
 
-`n-gage` is in every Next project, giving a standard set of `make` tasks and `ngage` CLI to help with setting up, building and deployments.
 
-<br clear="right">
+## Requirements
 
-## Getting started
+* Node 8.x
 
-Starting a new repo?  You can do the following:
+
+## Installation
+
+```sh
+git clone git@github.com:Financial-Times/n-gage.git
+cd n-gage
+make install
+```
+
+## Development
+
+### Testing
+
+In order to run the tests locally you'll need to run:
 
 ```sh
-mkdir my-new-project
-cd my-new-project
-npm init -y
-npm install --save-dev --no-package-lock @financial-times/n-gage
+make test
 ```
 
-then create a new `Makefile` file with the following:
+### Install from NPM
+
+```sh
+npm install --save @financial-times/n-gage
+```
+
+### Usage
+
+Create a new `Makefile` file with the following:
 
 ```make
-# n-gage bootstrapping logic
+# This task tells make how to 'build' n-gage. It npm installs n-gage, and
+# Once that's done it overwrites the file with its own contents - this
+# ensures the timestamp on the file is recent, so make won't think the file
+# is out of date and try to rebuild it every time
 node_modules/@financial-times/n-gage/index.mk:
-	npm install --no-save --no-package-lock @financial-times/n-gage
+	npm install --no-save @financial-times/n-gage
 	touch $@
 
+# If, by the end of parsing your `Makefile`, `make` finds that any files
+# referenced with `-include` don't exist or are out of date, it will run any
+# tasks it finds that match the missing file. So if n-gage *is* installed
+# it will just be included; if not, it will look for a task to run
 -include node_modules/@financial-times/n-gage/index.mk
 ```
 
-See [here](#bootstrapping) for more explanation of the bootstrapping logic.  You will want to add `unit-test`, `test`, `provision`, `smoke` and `deploy` tasks to the `Makefile`. See other, similar Next projects for ideas.
-
-## Git hooks
+### Make tasks
+
+| Task | Description |
+|-|-|
+| a11y                | Run automated accessibility tests |
+| build               | Build this repository |
```

Comment:
Does 'this' refer to the specific app that is using `ngage`? It reads like it is referring to `n-gage` itself, i.e. build the `n-gage` repository.

Maybe change mentions of 'this' to 'the', i.e. "Build the repository".

There are a few instances of this in this table

Reply:
yeah it refers to the repo where `n-gage` is installed. I'll update the descriptions.

Reply:
These don't seem to have been updated in the README **Make tasks** table.